mailpoet/assets/js/src/form_editor/components/form_settings/form_placement_options/below_pages.tsx

```diff
@@ -17,7 +17,7 @@ export function BelowPages(): JSX.Element {
       active={formSettings.formPlacement.belowPosts.enabled}
       label={MailPoet.I18n.t('placeFormBellowPages')}
       icon={BelowPageIcon}
-      onClick={(): void => showPlacementSettings('below_post')}
+      onClick={(): void => showPlacementSettings('below_posts')}
```

Comment:
In the end, it was OK as these constants were used only for the panel displaying and not connected in any way to the saved ones. I just unified it to avoid confusion.

Now [there is a union](https://github.com/mailpoet/mailpoet/pull/4123/files#diff-1c672ebda97fd1aa0d4c3eeff87dae023a7af17969bd3a4069b2c2762faca6a9R108) and that should check types as well (compared to no types before). I guess union is simpler and sufficient here so I'd maybe prefer to keep it when it works now. I didn't want to dig too deep into the old code, just wanted to make the type packages work.

Also considering https://codep2.wordpress.com/2022/03/30/typescript-avoid-enums/, it is a question of what direction we'll choose for MailPoet.